struct.c

```diff
@@ -1433,14 +1487,15 @@ rb_struct_eql(VALUE s, VALUE s2)
 
 /*
  *  call-seq:
- *     struct.length    -> integer
- *     struct.size      -> integer
+ *    size -> integer
  *
- *  Returns the number of struct members.
+ *  Returns the count of members.
```

Comment:
I think `number` is better than `count` here.  `count` should be used when something requires an action per thing being counted, whereas the size of a struct is always fixed per class.

Reply:
Done.